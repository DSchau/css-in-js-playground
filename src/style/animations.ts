import { keyframes } from 'glamor';

export const SLIDE_UP = keyframes({
  from: {
    bottom: '-7.5%'
  },
  to: {
    bottom: '7.5%'
  }
});
