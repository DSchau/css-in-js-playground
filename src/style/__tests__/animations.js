import * as ANIMATIONS from '../animations';

test('it contains a SLIDE_UP animation', () => {
  expect(ANIMATIONS.SLIDE_UP).toMatchSnapshot();
});
