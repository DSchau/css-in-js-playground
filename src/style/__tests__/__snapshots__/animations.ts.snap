// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it contains a SLIDE_UP animation 1`] = `"animation_bojjao"`;
